src/app/core/entity/entity-config.service.ts

```diff
@@ -28,9 +25,15 @@ export class EntityConfigService {
       );
     }
   }
+
+  public getEntityConfig(entityType: typeof Entity): EntityConfig {
```

Comment:
I think the type you want here would be `EntityConstructor` (defined in entity.ts)?

Reply:
The `EntityConstructor` type is used when new entities of a type have to be created. `typeof Entity` defines the class of an object and allows to access the static variables and is provided through the `Entity` [class](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/entity/entity.ts#L151). In this case only the static attribute `ENTITY_TYPE` is required and therefore `typeof Entity` provides the highest type-safety.